docker/clickhouse-builder/assets/arm64.compile.Dockerfile

```diff
@@ -15,34 +37,27 @@ RUN groupadd -r clickhouse --gid=101 \
         locales \
         wget \
         tzdata \
-    && wget --progress=bar:force:noscroll "https://github.com/tianon/gosu/releases/download/$gosu_ver/gosu-$(dpkg --print-architecture)" -O /bin/gosu \
+    && wget --progress=bar:force:noscroll "https://github.com/tianon/gosu/releases/download/$GOSU_VER/gosu-$(dpkg --print-architecture)" -O /bin/gosu \
     && chmod +x /bin/gosu
 
+COPY --from=builder ClickHouse/build/programs/clickhouse* /usr/bin/
```

Comment:
What does this do with symlinks? I've got a feeling if resolves them and hence will be very large.